src/hooks/useTokenAllowance.ts

```diff
@@ -8,16 +8,23 @@ import { calculateGasMargin } from 'utils/calculateGasMargin'
 
 import { useTokenContract } from './useContract'
 
-export function useTokenAllowance(token?: Token, owner?: string, spender?: string): CurrencyAmount<Token> | undefined {
+export function useTokenAllowance(
+  token?: Token,
+  owner?: string,
+  spender?: string
+): {
+  tokenAllowance: CurrencyAmount<Token> | undefined
+  isSyncing: boolean
+} {
   const contract = useTokenContract(token?.address, false)
 
   const inputs = useMemo(() => [owner, spender], [owner, spender])
-  const allowance = useSingleCallResult(contract, 'allowance', inputs).result
+  const { result, syncing: isSyncing } = useSingleCallResult(contract, 'allowance', inputs)
```

Comment:
`isSyncing` here too

Reply:
unclear what result means here?

Reply:
`useSingleCallResult` is a wrapper around the multicall library, which returns a `CallState` object that includes `syncing`, not `isSyncing`. I do not want to touch it, as it is not modified from the library.

NB: I originally used `syncing` and `pending` instead of `isSyncing` because it matched the syntax exported by `CallState`.